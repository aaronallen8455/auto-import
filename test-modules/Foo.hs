module Foo where

class Foo where
  f1 :: ()
  b2 :: ()

type Bar = ()

baz :: ()
baz = ()
