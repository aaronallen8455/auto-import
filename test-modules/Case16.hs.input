module Case16 where

import Foo (Foo(f1), baz)

main :: IO ()
main = f1 b2 Bar baz
