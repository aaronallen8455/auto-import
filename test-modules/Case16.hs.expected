module Case16 where

import Foo (Foo(f1, b2), baz, Bar)

main :: IO ()
main = f1 b2 Bar baz
